special-pages/pages/history/app/components/Sidebar.module.css

```diff
@@ -39,6 +39,23 @@
             background: var(--color-white-at-6);
         }
     }
+
+    &:last-child {
```

Comment:
I don't think it's always last-child, right? it's only this new 'sites' thing.

Reply:
@shakyShane Thank you! I initially thought 'Sites' would always be the last item, but I see we might add or move other items later. I've updated it in https://github.com/duckduckgo/content-scope-scripts/pull/1954/commits/8facc6254372de53e28c5bcf998a920b9b934165 to target 'Sites' directly.